pkg/install/billing_test.go

```diff
@@ -150,6 +211,21 @@ func TestCreateBillingEntry(t *testing.T) {
 					t.Errorf("Error want (%s), having (%s)", tt.wantError.Error(), err.Error())
 				}
 			}
+			if tt.validatee2eblob {
+				t.Log("Validating E2E Blob")
+				client, err := azstorage.NewBasicClient(env.E2EStorageAccountName(), env.E2EStorageAccountKey())
+				if err != nil {
+					t.Error("Error to initate az storage test client")
+				}
+				bs := client.GetBlobService()
+				containerName := "bill-" + location + "-rgname-clustername"
+				containerRef := bs.GetContainerReference(containerName)
+				blob := containerRef.GetBlobReference("billingentity")
+				existsAndDeleted, err := blob.DeleteIfExists(nil)
+				if !existsAndDeleted || err != nil {
+					t.Error("Error validating if the blob is present")
+				}
+			}
```

Comment:
This works well when using `azurite`, if we decide to keep it.

Reply:
I think this should be mocked. This creates unnecessary complexity around here for 4 API calls.